### PR TITLE
Edited PSR-15 files for small typos/grammar fixes.

### PR DESCRIPTION
**Pull request by DocFX**
Only non-structural changes, just a few punctuation/wording/grammar fixes.

### CODE REVIEW

File: accepted/PSR-15-request-handlers.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This document describes common interfaces for HTTP server request handlers
 that use HTTP messages as described by [PSR-7][psr7] or subsequent
 replacement PSRs.
 
-HTTP request handlers are a fundamental part of any web application. Server side
+HTTP request handlers are a fundamental part of any web application. Server-side
```
Comment (shadowhand) [inline accepted/PSR-15-request-handlers.md]: Not sure I agree with this change. While hyphenating might be more precise, I have seen it with and without the hyphen. Would this modification really make the document better?

File: accepted/PSR-15-request-handlers.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ specific HTTP message implementation.
 
 ### 1.4 Handling Exceptions
 
-It is RECOMMENDED that any application using middleware include a component
+It is RECOMMENDED that any application using middleware includes a component
```
Comment (shadowhand) [inline accepted/PSR-15-request-handlers.md]: This doesn't seem grammatically correct me.

Comment (Crell) [reply accepted/PSR-15-request-handlers.md]: English being the hodge podge of weirdness that it is, "application includes" or "applications include".  One or the other but not both gets an S.  (It's a XOR operation.)

Don't ask me, I don't understand this wonky language we use...

Comment (ashnazg) [reply accepted/PSR-15-request-handlers.md]: This one is straightforward... subject+verb is application+includes... both are singular.  A plural-ish prepositional phrase between the two is the usual blame for getting the verb wrong.

File: accepted/PSR-15-request-handlers.md (hunk 1 above)
Comment (DocFX) [reply accepted/PSR-15-request-handlers.md]: Just something I saw along the way by going quite far into typos.
Answer is here, I believe: https://en.wikipedia.org/wiki/Server-side

File: accepted/PSR-15-request-handlers.md (hunk 2 above)
Comment (DocFX) [reply accepted/PSR-15-request-handlers.md]: The real reason behind this is context. The next sentence uses singular and not plural. Was just for cohesion.

File: accepted/PSR-15-request-handlers.md (hunk 1 above)
Comment (DocFX) [reply accepted/PSR-15-request-handlers.md]: It's compound when used as a concept: "server-side".
It's separated when used as two words: "It's a concern that happens to be on the server('s) side.".

**Pull request merged**